lib/mix/lib/mix/tasks/format.ex

```diff
@@ -135,32 +164,64 @@ defmodule Mix.Tasks.Format do
   @manifest "cached_dot_formatter"
   @manifest_vsn 1
 
+  @doc """
+  Returns which features this plugin should plug into.
+  """
+  @callback features(Keyword.t()) :: [sigils: [atom()], extensions: [binary()]]
```

Comment:
What do you think about `inputs`, `sources`, `locations`, or `identifiers` as a replacement for `features`? For some reason, the term ["feature"](https://en.wikipedia.org/wiki/Software_feature) does not feel right to me 🙂

Reply:
I am not a big fan of `features` but I don't think any of the other ones make it justice. I thought maybe call it `plugin`, but I don't think it is better either. :(

Reply:
Oh! It could be `plug_in`.

Another alternative is to split `features` in two, `sigils` and `extensions`, but not a big fan of that option either :(